openbb_platform/obbject_extensions/charting/openbb_charting/charting.py

```diff
@@ -133,8 +134,13 @@ def get_params(self) -> ChartParams:
         ).replace("/", "_")[1:]
         if hasattr(ChartParams, charting_function):
             return getattr(ChartParams, charting_function)()
-        raise ValueError(
-            f"Error: No chart parameters are defined for the route: {charting_function}"
+
```

Comment:
It does raise, in the lines above, if there is no route.  After verifying the route exists, it checks ChartParams for that function; if it doesn't exist, then it will return the charting function's help. This allows parameters to be detailed in the code, outside of the charting extension.